### PR TITLE
Render teammate component on proxy

### PR DESCRIPTION
**Pull request by CodeWritingCow**
Please review my code. Thanks!

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static(__dirname + '/../public'));
+// app.use(express.static(__dirname + '/../client/dist'));
+
+app.get('/', (req, res, err) => {
+    res.send(data);
+});
+
```
Comment (WormJim) [inline server/app.js:14]: This Code above doesn't do anything special.  

If your intention is to send the public directory with all it's content, the code `app.use(express.static(__dirname + '/../public'));` is doing just that.  However, your path should be that you specify here isn't configured correctly.

When using the above syntax, and appending the path `'/../public'` to the `__dirname'  you are effectively extending the path to the `sever.js` file to `server.js/../public`.  which is not what you want.  You need to use the path module provided by Node.

File: server/app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+let port = process.env.PORT || 3017;
+app.listen(port, function () {
+    console.log(`Listening on port ${port}`);    
+});
+
+module.exports = app;
```
Comment (WormJim) [inline server/app.js:21]: A single page application will need to send back the proxy index.html file every time the path in the URL is changed.  
The way this proxy is set up will never handle any of the React Components JavaScript. This is a "dumb" proxy which should only renders the main `index.html` file.  Create a path for any route that you expect to be handled, which in turn returns that file.

If you'd like to use a Route in express to handle any direction to `/:id` there are two ways to achieve this.

1. create an `app.get` route to handle this path.  The route must return the index.html file
2. create a "Catch All" route which effectively will catch any path not specifically declared. The route must return the index.html file.  You can use the asterisk as a catch all. Example: `app.get('*', callback)`

File: public/index.html [hunk 3]
```diff
@@ -14,8 +14,8 @@ <h1>Proxy Server</h1>
     <script src="/lib/react-dom.development.js"></script>
     <!-- <script src="http://localhost:3016/client/dist/bundle.js"></script> -->
     <script src="http://localhost:3016/bundle.js"></script>
-    <script>
+    <!-- <script>
         ReactDOM.render(React.createElement(App), document.getElementById('app'));
-    </script>
+    </script> -->
 </body>
```
Comment (WormJim) [inline public/index.html:30]: Very Good catch. Get rid of the RectDom Render function.  Not needed.

File: server/app.js [hunk 4]
```diff
@@ -1,13 +1,14 @@
 const express = require('express');
 const app = express();
+// const path = require('path');
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(express.static(__dirname + '/../public'));
-// app.use(express.static(__dirname + '/../client/dist'));
+// app.use(express.static(path.join(__dirname, 'public')));
```
Comment (WormJim) [inline server/app.js:10]: This is great.  This is exactly what I was referring to in an earlier comment.  You can disregard that comment.

File: public/index.html [hunk 5]
```diff
@@ -3,19 +3,17 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Page Title</title>
+    <title>Proxy Page</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (WormJim) [inline public/index.html:7]: The title should emulate Trip Advisor.  That is what we are trying to create.

File: public/index.html [hunk 6]
```diff
@@ -3,19 +3,17 @@
...
     <link rel="stylesheet" type="text/css" media="screen" href="http://localhost:3016/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
 </head>
 <body>
-    <h1>Proxy Server</h1>
-    <div id="app"></div>
+    <div id="Details"></div>
+    <div id="events"></div>
     <script src="/lib/react.development.js"></script>
```
Comment (WormJim) [inline public/index.html:16]: Let's be consistent here.  I'd suggest making the id's Proper Case.  That way they are easily recognized. 

Also, I recall you using the word 'Categories' before.  I think that best explains what your component is and does.  It renders additional Categories of Attractions the purchaser may be interested in.

File: public/index.html [hunk 7]
```diff
@@ -3,19 +3,17 @@
...
     <script src="/lib/react-dom.development.js"></script>
-    <!-- <script src="http://localhost:3016/client/dist/bundle.js"></script> -->
+    <script src="http://localhost:3014/bundle.js"></script>
     <script src="http://localhost:3016/bundle.js"></script>
-    <!-- <script>
```
Comment (WormJim) [inline public/index.html]: Awesome that you removed the actual path to the bundle.js file.  Can you tell me why you don't need to use that?

Hint: it has something to do with `express.static()`

File: server/app.js (hunk 1 above)
Comment (WormJim) [reply server/app.js:14]: Ahh, you changed it.  Great!!!

**Review by WormJim — APPROVED** [review]

File: public/index.html (hunk 6 above)
Comment (CodeWritingCow) [reply public/index.html:16]: Thanks. Done!

**Pull request merged**